bin/node/runtime/src/lib.rs

```diff
@@ -813,7 +814,7 @@ impl pallet_contracts::Config for Runtime {
 	type DepositPerStorageItem = DepositPerStorageItem;
 	type RentFraction = RentFraction;
 	type SurchargeReward = SurchargeReward;
-	type MaxDepth = MaxDepth;
+	type CallStack = [pallet_contracts::Frame<Self>; 31];
```

Comment:
This changes relates to contracts/src/exec.rs module refactoring. Looks like call stack size previously was configured via `MaxDepth`